provider/lxd/environ_raw.go

```diff
@@ -67,8 +65,19 @@ func newRawProvider(spec environs.CloudSpec, local bool) (*rawProvider, error) {
 		prov, err := newLocalRawProvider()
```

Comment:
As we strangle this out, this is where we would move the default network and storage checks from the tools/lxdclient client.